transaction/builder/src/transaction_builder.rs

```diff
@@ -674,9 +699,7 @@ impl<FPR: FogPubkeyResolver> TransactionBuilder<FPR> {
 
     /// Return low level data to sign and construct transactions with external
     /// signers
-    pub fn build_unsigned<T: RngCore + CryptoRng, O: TxOutputsOrdering>(
-        mut self,
-    ) -> Result<UnsignedTx, TxBuilderError> {
+    pub fn build_unsigned<O: TxOutputsOrdering>(mut self) -> Result<UnsignedTx, TxBuilderError> {
```

Comment:
there was an unnecessary `RngCore + CryptoRng` generic parameter here, which I have removed in this revision